samples/openthread/coap_client_multiprotocol_ble/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+cmake_minimum_required(VERSION 3.13.1)
+
+set(NRF_SUPPORTED_BOARDS
+	nrf52840dk_nrf52840
+	nrf52833dk_nrf52833
+)
+
+find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
+
+project(openthread_ble_coap_client)
+
+FILE(GLOB app_sources src/*.c ../coap_client/src/coap_client_utils.c)
```

Comment:
Have you considered having a single sample which is configurable with overlay file to turn multiprotocol on? Just like zigbee did in their light switch? That would allow to duplicate even less (core prj.conf, common parts of the main file).

Reply:
Ok, after the code refactoring main CoAP Client file looks clear enough to be merged with multiprotocol extension. Done.